src/main.rs

```diff
@@ -329,11 +326,18 @@ fn adjust_ulimit_size(opts: &Opts) -> u32 {
     rlim.try_into().unwrap()
 }
 
+// Rlimit does not support Windows
+// set to 1000 if Windows is used
+#[cfg(target_os = "windows")]
```

Comment:
Not too sure on the idea of having 2 functions with the same name, @bernardoamc what do you think?

Reply:
I'm not sure what the convention is, it makes me a little itchy as well to have two functions with the same name. I think though that this is the way to keep the compiler from freaking out, as checking `cfg(target_os = "windows")` in a conditional doesn't cause the compiler to ignore the code. At least, as much as I can tell. I agree though it's a little off-putting, I just went with the quick-and-dirty patch that worked for me.

Reply:
If I'm not mistaken we can do an `if/else` with `cfg!(target_os = "windows")`, but I don't have a strong opinion regarding functions with the same name but that get stripped out during compilation. :)